demo/java/build.gradle

```diff
@@ -0,0 +1,53 @@
+plugins {
+    id 'java'
+}
+
+group 'org.hyperledger'
+version '0.8.0'
+
+sourceCompatibility = 1.8
+
+repositories {
+    mavenCentral()
+    maven {
+        url "http://13.125.219.189/repository/maven-public/" // skt maven
+    }
+}
+
+dependencies {
+    // wrapper for vcx library
+    implementation 'com.evernym:vcx:0.8.2-sktston-SNAPSHOT' // use maven repository
```

Comment:
I'll merge it as is, but it would be cool if you'd add github action to publish java wrapper and then use that in this java demo